lib/db/mysql/index.js

```diff
@@ -474,6 +476,11 @@ MysqlStore.prototype = {
    * @returns {Promise}
    */
   deleteActiveClientTokens: function deleteActiveClientTokens(clientId, uid) {
```

Comment:
Possible bike-shedding alert, but does this function deserve a more generic name now that it also deletes codes as well as tokens, or is the distinction between the two sufficiently blurry that "tokens" works well as a collective term for both entities?

Reply:
I went with 'deleteClientAuthorization' which seems a little more generic - we delete anything that authorizes the client to do things on the user's account.